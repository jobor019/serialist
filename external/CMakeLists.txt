
add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/Catch2 EXCLUDE_FROM_ALL)
add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/expected EXCLUDE_FROM_ALL)
#add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/JUCE EXCLUDE_FROM_ALL)
add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/magic_enum EXCLUDE_FROM_ALL)

