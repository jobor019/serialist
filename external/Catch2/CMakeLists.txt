
FIND_PACKAGE(Catch2 REQUIRED)