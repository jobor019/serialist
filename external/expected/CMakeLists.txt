# Simplistic CMakeLists.txt for expected to avoid certain includes in the source CMakeLists.txt

add_library(expected INTERFACE)
target_include_directories(expected INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/expected/include)

add_library(tl::expected ALIAS expected)